Misc/NEWS.d/next/Core and Builtins/2024-01-17-05-09-32.gh-issue-112354.Run9ko.rst

```diff
@@ -0,0 +1,2 @@
+The ``END_FOR`` instruction now pops only one value. This is to better
```

Comment:
With this change, documentation for ``END_FOR`` should be updated.
Currently it says: "Removes the top two values from the stack."